.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
```

Comment:
We seem to be using 4 spaces for certain JSON files, so perhaps let's account for that.

Reply:
>We seem to be using 4 spaces for certain JSON files, so perhaps let's account for that.

@P0lip I can think of three options:
 - Include in the PR normalized versions (to two spaces wide indentation) of those files (under `/src/meta`, for instance) 
  - If those indentations have to be preserved (to ease diffing comparison with an external source of truth for instance), add another `.editorconfig` file under this hierarchy specifying that `.json` files here and below are expected to be indented with 4 spaces
 - Update the root `.editorconfig` file to replace the `*` with something more restrictive which would explicitly not consider json (eg. `[*.{ts,js,yaml}]`)

Reply:
Let's go with `[*.{ts,js,yaml}]` so we can avoid doing anything funky with Markdown files (like trimming intentional trailing whitespace).

Reply:
Yeah.
That said, we can still have charset set to utf-8 for all file types.

Re YAML - I'd stick to indentation only, as trailing spaces and newlines might be intentional in some cases.

Reply:
@philsturgeon @P0lip Latest proposal should encompass all those requirements above